Open-Rival/src/Resources.cpp

```diff
@@ -109,74 +115,36 @@ namespace Rival {
                 Palette::createPaletteTexture());
     }
 
-    void Resources::initUnitSpritesheets() {
+    void Resources::initBuildingSpritesheets() {
+        buildingSpritesheets = std::make_unique<std::map<BuildingType, Spritesheet>>();
+        int nextIndex = txIndexBuildings;
+
+        auto createSpreadsheets = [&](int first, int last) {
```

Comment:
Should be `createSpritesheets` ?

Reply:
I thought you might need some calculations. :P Corrected.